src/rust/ide/src/model/project.rs

```diff
@@ -24,6 +24,7 @@ use uuid::Uuid;
 #[automock]
 pub trait API:Debug {
     /// Project's name
+    // TODO [mwu] This should return Rc<ReferentName>.
     fn name(&self) -> ImString;
```

Comment:
Why `ReferentName` is not immutable by default (Why it doesn't embed `Rc` inside?)

Reply:
It was introduced as a replacement for non-immutable `String`. Types in double representation module generally expect that Rc will be used on them, rather than built it in.